packages/web3-eth/src/rpc_method_wrappers.ts

```diff
@@ -1068,42 +1085,34 @@ export function sendTransaction<
 		(resolve, reject) => {
 			setImmediate(() => {
 				(async () => {
-					try {
-						let transactionFormatted = formatTransaction(
-							{
-								...transaction,
-								from: getTransactionFromOrToAttr('from', web3Context, transaction),
-								to: getTransactionFromOrToAttr('to', web3Context, transaction),
-							},
-							ETH_DATA_FORMAT,
-						);
-
-						if (web3Context.handleRevert) {
-							// eslint-disable-next-line no-use-before-define
-							await getRevertReason(
+					let transactionFormatted = formatTransaction(
+						{
+							...transaction,
+							from: getTransactionFromOrToAttr('from', web3Context, transaction),
+							to: getTransactionFromOrToAttr('to', web3Context, transaction),
+						},
+						ETH_DATA_FORMAT,
+					);
+
+					if (
+						!options?.ignoreGasPricing &&
+						isNullish(transactionFormatted.gasPrice) &&
+						(isNullish(transaction.maxPriorityFeePerGas) ||
+							isNullish(transaction.maxFeePerGas))
+					) {
+						transactionFormatted = {
+							...transactionFormatted,
+							// TODO gasPrice, maxPriorityFeePerGas, maxFeePerGas
+							// should not be included if undefined, but currently are
+							...(await getTransactionGasPricing(
+								transactionFormatted,
 								web3Context,
-								transactionFormatted as TransactionCall,
-							);
-						}
-
-						if (
-							!options?.ignoreGasPricing &&
-							isNullish(transactionFormatted.gasPrice) &&
-							(isNullish(transaction.maxPriorityFeePerGas) ||
-								isNullish(transaction.maxFeePerGas))
-						) {
-							transactionFormatted = {
-								...transactionFormatted,
-								// TODO gasPrice, maxPriorityFeePerGas, maxFeePerGas
-								// should not be included if undefined, but currently are
-								...(await getTransactionGasPricing(
-									transactionFormatted,
-									web3Context,
-									ETH_DATA_FORMAT,
-								)),
-							};
-						}
+								ETH_DATA_FORMAT,
+							)),
+						};
+					}
 
```

Comment:
Removed from `try/catch` to make `transactinFormatted` available to `getTransactionError` in `catch`